src/components/OptionsAccordion/OptionsAccordion.module.scss

```diff
@@ -70,3 +70,8 @@ $containerRadius: var(--space-sm);
 .sectionTitle {
   padding: var(--space-sm) 0px;
 }
+
+.panelList {
+  max-height: 50vh;
```

Comment:
could we use something other than `vh`? `px`? I find that is usually breaks/is hard to work with

Reply:
Rem?